### PR TITLE
Upgrade neutron agent together with nova-compute package (SOC-11031)

### PR DESCRIPTION
**Pull request by jsuchome**
In some cases we start losing network connectivity to VM's when
all nova-compute services are upgraded and live-migration is being
performed. Upgrading neutron agents on the compute nodes seems to help.

Just a note: this is consistent with the code in 4.0 branch, where we also upgraded nova-compute on all nodes during the upgrade...

### CODE REVIEW

File: chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb [hunk 1]
```diff
@@ -97,7 +97,7 @@ if [ $ret != 0 ] ; then
     exit $ret
 fi
 
-log "Restarting services for Rocky"
```
Comment (skazi0) [inline chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb]: why did you remove this log line?

Comment (jsuchome) [reply chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb]: Because it's no longer true (no Rocky when upgrading to SOC9) ... and anyway it does not have much sense, the script actions are logged already

Comment (jsuchome) [reply chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb]: OK, I might be wrong about Rocky and SOC9. But I still think the line does not bring any value (esp. when services are not _always_ restarted, there are further checks, see below)

**Comment by jsuchome** [comment]
It seems it has proved to be useful, removing wip flag.
@jgrassler, chance to veto?

**Comment by skazi0** [comment]
Tested in the scale lab and it seems to solve our conn-loss problem.
